Src/Campus.Master.API/UI/src/app/shared/services/registration.service.ts

```diff
@@ -11,6 +12,11 @@ export class RegistrationService {
   constructor(private httpClient: HttpClient) { }
 
   registerUser(user: RegisterUser): Observable<StateTransfer> {
-    return this.httpClient.post<StateTransfer>(environment.createProfilePath, user);
+    const uri = environment.createProfilePath;
```

Comment:
It's better to keep API routes inside service, so the environment settings won't turn into **GOD OBJECT**. Refactor this later plz

Reply:
Got it